tools/bridge/bridge/main.py

```diff
@@ -224,6 +224,23 @@ def stop(pool, timeout):
         os._exit(os.EX_SOFTWARE)
 
 
+def reload_logging_config(config_path):
+    gevent.getcurrent().name = "sighup"
+    logger.info(
+        f"Handling sighup, trying to reload the logging configuration from {config_path}"
+    )
+    try:
+        config = load_config(config_path)
+        configure_logging(config)
+        logger.info("Logging has been reconfigured")
+    except Exception as err:
+        # this is a signal handler. make sure we don't die as this
+        # will raise the error in the main greenlet.
```

Comment:
I don't understand this comment (I'm missing some required background though). Maybe clarify with `... don't die so that the error is raised in the main greenlet`, or `don't die as this would raise ...`, depending on what's true.